support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -251,6 +264,26 @@ function Heading({
 		);
 	}
 
+	if (isPaperOrGuardianWeekly) {
+		const getPrintTitle = (productOption?: string) => {
```

Comment:
The parameter `productOption` is in fact `ratePlanKey` which is passed in here: `getPrintTitle(ratePlanKey)`, I think it's a bit confusing creating a new variable name (`productOption`) for something that's referred to as `ratePlanKey` everywhere else. 

We don't actually need to pass `ratePlanKey` to `getPrintTitle` as it's available/defined already in the parent scope of `getPrintTitle`, so you could just use `ratePlanKey` in the `switch`.